### PR TITLE
[1LP][RFR] New REST API test for ordering catalog item

### PR DESCRIPTION
**Pull request by mkoura**
... as a non-admin user.

{{pytest: -v --long-running -k test_user_item_order cfme/tests/services/test_rest_services.py}}

### CODE REVIEW

File: cfme/tests/services/test_rest_services.py [hunk 1]
```diff
@@ -5,12 +5,17 @@
...
+from cfme.rest.gen_data import _creating_skeleton
+from cfme.rest.gen_data import blueprints as _blueprints
+from cfme.rest.gen_data import copy_role
 from cfme.rest.gen_data import dialog as _dialog
-from cfme.rest.gen_data import service_data as _service_data
+from cfme.rest.gen_data import groups
+from cfme.rest.gen_data import orchestration_templates as _orchestration_templates
 from cfme.rest.gen_data import service_catalogs as _service_catalogs
+from cfme.rest.gen_data import service_data as _service_data
 from cfme.rest.gen_data import service_templates as _service_templates
```
Comment (mkoura) [inline cfme/tests/services/test_rest_services.py]: Thanks, I'll do it in some other PR. I'm happy this one passed PRT and don't want to re-rest :) I'm touching this file in other PRs anyway.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/services/test_rest_services.py (hunk 1 above)
Comment (mkoura) [inline cfme/tests/services/test_rest_services.py]: needed to rebase anyway :(

File: cfme/tests/services/test_rest_services.py [hunk 2]
```diff
@@ -868,6 +875,81 @@ def test_delete_catalog_from_collection(self, appliance, service_catalogs):
...
+        wait_for(_order_finished, num_sec=180, delay=10)
+
+        service_name = re.search(
+            r'\[({}[0-9-]*)\] '.format(new_template.name), service_request.message).group(1)
+        # this fails if the service with the `service_name` doesn't exist
+        new_service = appliance.rest_api.collections.services.get(name=service_name)
+
+        @request.addfinalizer
+        def _finished():
+            new_service.action.delete()
```
Comment (quarckster) [inline cfme/tests/services/test_rest_services.py]: **OPTIONAL**
```python
request.addfinalizer(new_service.action.delete)
```

**Pull request merged**